inst/themer-demo/app.R

```diff
@@ -329,5 +389,31 @@ shinyApp(
       )
     })
 
+    tips_data <- reactive({
```

Comment:
Is it worth conditionally including this block when not in legacy mode?

Reply:
This code won't run if the `Dashboard` tab isn't included on the page (because of `outputOptions(suspendWhenHidden = TRUE)`, so not really.